### PR TITLE
Add language requirement for all pathauto patterns

### PR DESCRIPTION
**Pull request by ben-hamelin**
## Summary

Adds the "English" language config for pathauto patterns.
Removes 3 base field config overrides from ecms_promotions that were causing errors on install.

## Metadata

| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | yes
| Did you perform a self review first? | yes
| Documentation reflects changes? | no
| `CHANGELOG` reflects changes? | yes
| Unit/Functional tests reflect changes? | no
| Did you perform browser testing? | yes
| Risk level | Low
| Relevant links | https://thinkoomph.jira.com/browse/RIG-198

### CODE REVIEW

File: ecms_base/features/custom/ecms_event/config/install/pathauto.pattern.events.yml [hunk 1]
```diff
@@ -2,20 +2,31 @@ langcode: en
...
   module:
+    - language
     - node
 id: events
 label: Events
 type: 'canonical_entities:node'
 pattern: 'events/[node:title]'
 selection_criteria:
-  683b1f5f-b16e-4486-a1a4-766a8f95b390:
+  2b91bbb8-ae1d-4944-98bf-d6b628a417f1:
```
Comment (pfrilling) [inline ecms_base/features/custom/ecms_event/config/install/pathauto.pattern.events.yml:12]: Are these UUID's taken from the Covid site? What happens on a different site? Will the changes cause issues on another site?

I notice these changes in all of the content types.

**Review by pfrilling — COMMENTED** [review]
I think the changes look fine. I did have a question on the UUID's changing for each config.

File: ecms_base/features/custom/ecms_event/config/install/pathauto.pattern.events.yml (hunk 1 above)
Comment (ben-hamelin) [reply ecms_base/features/custom/ecms_event/config/install/pathauto.pattern.events.yml:12]: I saw that update too - and no, these were exported as features from the local development site. I looked at the rihavens site, and it had the "old" uuid too (for the page pattern), i.e. "4532149e-106e-4e15-b944-14a7007a3011"

https://rihavens.riecms.acsitefactory.com/admin/config/development/configuration/single/export

I'm wondering if these are simply arbitrary, based on some combination of the settings? I would say let's just test them out on stage after the next deploy to ensure things are functional.

**Comment by ben-hamelin** [comment]
@pfrilling for your feedback on the uuid

**Review by pfrilling — APPROVED** [review]
🚀

**Pull request merged**